scipy/sparse/linalg/_eigen/lobpcg/tests/test_lobpcg.py

```diff
@@ -533,7 +533,7 @@ def test_maxit():
     assert_allclose(np.shape(r_h), np.shape(np.asarray(r_h)))
 
 
-@pytest.mark.slow
+@pytest.mark.xslow
```

Comment:
Another option would be to @parametrize rather than looping and reduce the number of cases.